.github/workflows/build-core-template.yml

```diff
@@ -23,11 +23,16 @@ on:
         type: string
         required: false
         default: '[{ "zksolc": ["1.3.14", "1.3.16", "1.3.17", "1.3.1", "1.3.7", "1.3.18", "1.3.19", "1.3.21"] } , { "zkvyper": ["1.3.13"] }]'
+      en_alpha_release:
+        description: 'Flag that determins if EN release should be marked as alpha'
+        type: boolean
+        required: false
+        default: false
 jobs:
   build-images:
     name: Build and Push Docker Images
     env:
-      IMAGE_TAG_SUFFIX: ${{ inputs.image_tag_suffix }}
+      IMAGE_TAG_SUFFIX: ${{ format('{0}{1}', inputs.image_tag_suffix, (inputs.en_alpha_release && matrix.components == 'external-node') && '-alpha' || '') }}
```

Comment:
Do we really need `format()` func here?
I guess we can use simpler structure like:
`IMAGE_TAG_SUFFIX: ${{ inputs.image_tag_suffix }}${{ (inputs.en_alpha_release && matrix.components == 'external-node') && '-alpha' || '' }}`